core/controllers/admin.py

```diff
@@ -228,11 +239,20 @@ def get(self) -> None:
         topic_summary_dicts = [
             summary.to_dict() for summary in topic_summaries]
 
-        config_properties = config_domain.Registry.get_config_property_schemas()
-        # Removes promo-bar related configs as promo-bar is handled by
+        platform_params_dicts = (
+            feature_services.
+            get_all_platform_parameters_except_feature_flag_dicts())
+        # Removes promo-bar related platform params as promo-bar is handled by
         # release coordinators in /release-coordinator page.
-        del config_properties['promo_bar_enabled']
-        del config_properties['promo_bar_message']
+        platform_params_dicts = [
+            param for param in platform_params_dicts
+            if (
+                param['name'] != 'promo_bar_enabled' and
```

Comment:
Make a list of constants: PLATFORM_PARAMS_TO_SHOW_IN_RC_PAGE -- and use that only. Don't hardcode strings in the middle of the code.

Reply:
Hey @seanlip,
Just a thought: So as per the requirement of the project I have to add a `list` variable each for feature-flag and platform-parameter(#18355: Currently a draft as it also contains the changes of this PR), now if we add another variable then it will be a bit hard to update them when we add a new platform parameter. As an example, if we add a new platform-parameter then first it needs to be added to the enum class we maintain for all the platform params in the [platform-parameter-list](https://github.com/oppia/oppia/blob/develop/core/domain/platform_parameter_list.py#L29) then we will require to update the list I will be introducing for the platform params and if we introduce this new variable for the RC page then that needs to be updated as well. Also, I think we cannot create this PLATFORM_PARAMS_TO_SHOW_IN_RC_PAGE variable in the constants.ts file as we plan to use the enum class everywhere we reference the platform parameter.
Maybe instead of hardcoding it as strings I can use the enum class to reference the promo-bar platform param, let me know what do you think.

Thanks

Reply:
I'm confused about this, I'm just suggesting making a list to contain the two constants in this "if" condition that shows why these platform parameters are special-cased. You don't need to include it in the frontend or in constants.ts. This is just a backend constant at the top of the current file which is done to avoid having literal strings in the code.

Does that clarify?

Reply:
Ah!, okay. I somehow misunderstand, haha, sorry.
Done